configureBuild.sh

```diff
@@ -385,5 +385,7 @@ configure_build() {
   setWorkingDirectory
   configureMacFreeFont
   setMakeArgs
-  setBootJdk
+  if [ "${BUILD_CONFIG[USE_DOCKER]}" == false ] ; then
```

Comment:
I think from looking at the other changes, USE_DOCKER is either "docker" or "podman", and not "false"?
Which leads to the point that the variable name is confusing...?
Maybe we should add a new variable, say "CONTAINER_CMD" (docker|podman), and if we need a boolean, call that "USE_CONTAINER" (true|false) ??

Reply:
Thanx!

both variables must be renamed as last stage of this PR (I had nto yet done it to avoid broken rebases).

It is not visdibel in changeset, but defautl value of BUILD_CONFIG[USE_DOCKER] is really false.:
https://github.com/adoptium/temurin-build/blob/27b8c5ce58ae74d2343866ef63d28262ce83b4b5/sbin/common/config_init.sh#L579

Reply:
See also https://github.com/adoptium/temurin-build/pull/3796#discussion_r1593965733 and  https://github.com/adoptium/temurin-build/pull/3796#discussion_r1593972520